src/news/__tests__/newsActions.test.ts

```diff
@@ -0,0 +1,58 @@
+import fetchMock from 'fetch-mock'
+import configureMockStore from 'redux-mock-store'
+import { ThunkDispatch } from 'redux-thunk'
+import thunkMiddleware from 'redux-thunk'
+import { API, GlobalState } from 'types';
+import api from '../../api';
+import { getNewsData, NEWS_RECEIVED_ACTION, NewsReceivedAction, receiveNewsAction } from '../newsActions'
+import { News } from '../types'
+
+
+describe('actions', () => {
+
+    describe('synchronous actions', () => {
+        it('receiveNewsAction should create a news received action', () => {
+            const newsItems: News[] = [{ url: '', headline: '', datetime: '', source: '' }];
+            const expectedAction = {
+                type: NEWS_RECEIVED_ACTION,
+                newsItems
+            }
+
+            expect(receiveNewsAction(newsItems)).toEqual(expectedAction)
+        })
+    })
+
+    describe('asynchronous actions', () => {
+        const fetch = require('jest-fetch-mock');
+        jest.setMock('node-fetch', fetch);
+        type ThunkDispatchNewsReceivedAction = ThunkDispatch<GlobalState, API, NewsReceivedAction>
+        type Store = GlobalState | ThunkDispatchNewsReceivedAction
+
+        afterEach(() => {
+            fetchMock.reset()
+            fetchMock.restore()
+        })
+
+        it('getNewsData should  dpstach an async news received action', () => {
+            const newsData = { url: 'us', headline: 'us', datetime: 'us', source: 'us' };
+            // TODO ML 05/10 should move this one to api.test.ts for example 
+            fetchMock.getOnce('https://api.iextrading.com/1.0/stock/aapl/news/last/5', { body: { news: [newsData] }, headers: { 'content-type': 'application/json' } });
```

Comment:
Completely unsure about this, but I don't think mocking `fetch` is necessary here? I think you could just mock the `api.getNews()` function to return a fixed object.